charts/README.md

```diff
@@ -23,59 +25,57 @@ Downloading zookeeper from repo http://storage.googleapis.com/kubernetes-charts-
 Deleting outdated charts
 ```
 
-Note: in released versions of Zenko and the Chart, these upstream Charts will
-be embedded in the package created by `helm package`.
+**Note:** In released versions of Zenko and the Chart, these upstream charts
+are embedded in the package created by `helm package`.
 
-Then, assuming you have an Ingress Controller running, run something like
+Then, assuming you have an ingress controller running, run something like:
 
 ```shell
 $ helm install --name zenko --set ingress.enabled=true --set ingress.hosts[0]=zenko.local --set cloudserver-front.endpoint=zenko.local zenko
 ```
 
-See `values.yml` for more options.
+One noteworthy option is to enable integration with the [Orbit] management UI
+(disabled by default) and set a custom Orbit API endpoint (for development
```

Comment:
Not your fault, but the Orbit management is now enabled by default. Could we swap the wording around? So "option is to disable integration...(enabled by default)"